### PR TITLE
Issue #SB-19202 Add Chat bot to portal service

### PR DESCRIPTION
**Pull request by rajeevsathish**

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 1]
```diff
@@ -95,3 +95,5 @@ sunbird_collectionEditorURL=collection-editor/index.html
 sunbird_contentEditorURL=content-editor/index.html
 sunbird_genericEditorURL=generic-editor/index.html
 sunbird_keycloak_public_key={{core_vault_sunbird_sso_publickey}}
+sunbird_bot_configured=true
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:98]: is this going to be mandatory from 3.0 ? i mean for adopters ?

Comment (rajeevsathish) [reply ansible/roles/stack-sunbird/templates/sunbird_player.env:98]: if this config is not given or given as false then the bot will be disabled

Comment (SMYALTAMASH) [reply ansible/roles/stack-sunbird/templates/sunbird_player.env:98]: Then this need to be variablized @rajeevsathish ,

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 2]
```diff
@@ -95,3 +95,5 @@ sunbird_collectionEditorURL=collection-editor/index.html
 sunbird_contentEditorURL=content-editor/index.html
 sunbird_genericEditorURL=generic-editor/index.html
 sunbird_keycloak_public_key={{core_vault_sunbird_sso_publickey}}
+sunbird_bot_configured={{sunbird_bot_configured}}
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:98]: @rajeevsathish please keep a sane default value, if the bot is going to be inbuilt module keep it as `true`, else if it's an external module `false`

**Comment by rjshrjndrn** [comment]
> Comments to be resolved

@keshavprasadms LGTM

**Pull request merged**